zbar/video/dshow.c

```diff
@@ -1000,9 +1031,11 @@ static int dshow_determine_formats(zbar_video_t* vdo)
                     vdo->formats[n] = fmt;
                     ++n;
                 }
+                {
                 resolution_t resolution = { bih->biWidth, bih->biHeight };
```

Comment:
Please don't add a `{` here just due to the var declaration. That makes the C file a lot harder to review. From `zbar/misc.h`:

```
struct resolution_s {
    long cx;
    long cy;
};
typedef struct resolution_s resolution_t;
```

So, you can simply declare `resolution_t resolution` just after the function prototype, and then use here:
```
resolution.cx = bih->biWidth;
resolution.cy = bih->biHeight;
```

Reply:
OK